RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+18.6
+-----
+- [*] Change text color in badge view in order products list. [https://github.com/woocommerce/woocommerce-ios/pull/12630]
```

Comment:
super nit: maybe we can mention "dark mode" as a "fix" since this change doesn't affect the light mode